ios/RCTMGL-v10/RCTMGLMapView.swift

```diff
@@ -283,6 +285,9 @@ extension RCTMGLMapView {
         let event = RCTMGLEvent(type:.cameraChanged, payload: self.buildStateObject());
         self.fireEvent(event: event, callback: self.reactOnMapChange)
       }
+        
+        self.resetWasGestureIndicators();
```

Comment:
```suggestion
        self.wasGestureActive = self.isGestureActive;
```

Do we need reset here?! It's enough to reset in mapIdle isn't it?!